### DISCUSSION TITLE
next/image broken on mobile

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [vercel/next.js#38285] [Help] [answered]
I use a `next/image` component in my project, but on mobile the image disappears, it does not even take up the reserved space
repro: https://stackblitz.com/edit/nextjs-vjxdw5?file=pages/index.js
![image](https://user-images.githubusercontent.com/61914365/177071874-fb2b14f3-ff8c-425a-a452-684ac3d53ae4.png)

**Comment by valcosmos** [comment, marked as answer]
Hello, I tried your example, but I found that this may not be a problem of next/image, it may be a problem of css, I commented some classes above the parent element, and the picture is displayed normally.

Hope it can helps you.

![image](https://user-images.githubusercontent.com/53572196/177093760-9996baf3-5eb7-4bd6-9760-dac671eaddb5.png)

![image](https://user-images.githubusercontent.com/53572196/177093922-de99e510-3cba-4665-a173-4e58ccd337de.png)

**Reply by Chandler-Zhu** [reply]
That's interesting, maybe I'll just use `flex-col` on mobile. Thanks for your time

**Reply by valcosmos** [reply]
It is OK, and if it helps u, u might be able to mark it as the answer.😊
